### PR TITLE
OpenROAD Main README.md update

### LINKED ISSUE #1953: Let's include some images in the README
The README should include some statistics and pictures about the number of number of successful tape outs.

We should also include pictures of the GUI. These will serve as draws to get people involved.

### PR DESCRIPTION
**Pull request by vijayank88**
It resolves #1953

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,119 @@
...
+
+Two main flow controllers are supported by the
+[OpenROAD](https://github.com/The-OpenROAD-Project/OpenROAD)
+project repository:
+
+[OpenRoad-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts) -
+Supported by the OpenROAD project
+
+[OpenLane](https://github.com/The-OpenROAD-Project/OpenLane) -
+Supported by [Efabless](https://efabless.com/)
```
Comment (vvbandeira) [inline README.md:32]: Make a bullet list

File: README.md [hunk 2]
```diff
@@ -2,6 +2,119 @@
...
+turnaround from RTL-GDSII for design exploration and physical design
+implementation.
+
+![rtl2gds.webp](./docs/images/rtl2gds.webp)
+
+The OpenROAD GUI is a powerful visualization, analysis and debugging
+tool with a customizable tcl interface.
+
+Figures show GUI views for various flow stages including post-routed
+timing, placement congestion, CTS.
```
Comment (vvbandeira) [inline README.md:44]: Add a `## GUI` subsection and join these two paragraphs.

File: README.md [hunk 3]
```diff
@@ -2,6 +2,119 @@
...
+Openroad-flow-scripts supports several public and private PDKs
+including:
+
+## Open Source PDKs
+
+`Skywater130` - 130nm
+
+`Nangate45` - 45nm
+
+`ASAP7` - Predictive FinFET 7nm
```
Comment (vvbandeira) [inline README.md:73]: Remove empty lines and make a bullet list.

File: README.md [hunk 4]
```diff
@@ -2,6 +2,119 @@
...
+
+`GF12` - 12nm
+
+`GF180` - 180nm
+
+`Intel22` - 22nm
+
+`Intel16` - 16nm
+
+`TSMC65` - 65nm
```
Comment (vvbandeira) [inline README.md:94]: Remove empty lines and make a bullet list.

File: README.md [hunk 5]
```diff
@@ -2,6 +2,119 @@
...
+
+`Nangate45` - 45nm
+
+`ASAP7` - Predictive FinFET 7nm
+
+## Proprietary PDKs
+
+These PDKS are supported in OpenRoad-flow-scripts only. They are used to
+test and calibrate OpenROAD against commercial platforms and ensure good
+QoR.The PDKs and platform-specific files for these kits cannot be
```
Comment (vvbandeira) [inline README.md:79]: Missing space `QoR. The`

File: README.md [hunk 6]
```diff
@@ -2,6 +2,119 @@
...
+Supported by [Efabless](https://efabless.com/)
+
+The OpenROAD flow delivers an autonomous, no-human-in-the-loop, 24 hr
+turnaround from RTL-GDSII for design exploration and physical design
+implementation.
+
+![rtl2gds.webp](./docs/images/rtl2gds.webp)
+
+The OpenROAD GUI is a powerful visualization, analysis and debugging
+tool with a customizable tcl interface.
```
Comment (vvbandeira) [inline README.md:41]: `tcl` -> `Tcl`

File: README.md [hunk 7]
```diff
@@ -2,6 +2,119 @@
...
+
+# PDK Support
+
+The OpenROAD application is PDK independent. However, it has been tested
+and validated with specific PDKs in the context of various flow
+controllers.
+
+OpenLane supports Skywater130.
+
+Openroad-flow-scripts supports several public and private PDKs
```
Comment (vvbandeira) [inline README.md:64]: OpenROAD

File: README.md [hunk 8]
```diff
@@ -2,6 +2,119 @@
...
+mixed-signal design flows.
+
+OpenROAD users range from hardware designers, industry collaborators,
+enthusiasts, academia and researchers.
+
+Two main flow controllers are supported by the
+[OpenROAD](https://github.com/The-OpenROAD-Project/OpenROAD)
+project repository:
+
+[OpenRoad-flow-scripts](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts) -
```
Comment (vvbandeira) [inline README.md:28]: OpenROAD

**Review by vvbandeira — CHANGES_REQUESTED** [review]

**Comment by vvbandeira** [comment]
Thanks, @vijayank88 LGTM. @QuantamHD do you have any feedback on this and can you confirm this meets the criteria to close the issue?

**Comment by QuantamHD** [comment]
LGTM otherwise! Looks great

**Review by vvbandeira — APPROVED** [review]

**Pull request merged**